src/Mod/Draft/draftguitools/gui_trimex.py

```diff
@@ -120,8 +120,7 @@ def proceed(self):
             self.extrudeMode = True
             self.ghost = [trackers.ghostTracker([self.obj])]
             self.normal = self.obj.Shape.Faces[0].normalAt(0.5, 0.5)
-            for v in self.obj.Shape.Vertexes:
-                self.ghost.append(trackers.lineTracker())
+            self.ghost += [trackers.lineTracker()] * len(self.obj.Shape.Vertexes)
```

Comment:
@chennes 
self.ghost + = [trackers.lineTracker ()] * len (self.obj.Shape.Vertexes)
add references to the same lineTracker object and here we need references to several lineTrackers (one for each vertex).
To recreate the problem, draw a face, extrude it using the trimex command (pressing the z key).
It will be observed that the ghost line is generated for only one vertex.
At the end of the command, several errors are reported due to the inconsistency in the number of trackers explained above.
Is there another way to make the analyzer not complain?

Reply:
does the analyzer complain even adding an underscore to the unused loop variables?

Reply:
No, the underscore convention will work, as will naming the variable "unused".

Reply:
ok, we can add this to the coding conventions file.